src/main/java/appeng/client/ClientHelper.java

```diff
@@ -199,11 +182,6 @@ public RayTraceResult getRTR()
 	@Override
 	public void postInit()
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1186)